examples/tutorials/core_api_mnist/adaptive.yaml

```diff
@@ -0,0 +1,34 @@
+name: coreapi_mnist_tutorial_adaptive_search
+description: Use an adaptive ASHA search to optimize a hyperparameter
+hyperparameters:
+  global_batch_size: 64
+  learning_rate:
+    type: double
+    minval: .0001
+    maxval: 1.0
+  n_filters1:
+    type: int
+    minval: 8
+    maxval: 64
+  n_filters2:
+    type: int
+    minval: 8
+    maxval: 72
+  dropout1:
+    type: double
+    minval: .2
+    maxval: .8
+  dropout2:
+    type: double
+    minval: .2
+    maxval: .8
+max_restarts: 0
+records_per_epoch: 60000
+searcher:
+  name: adaptive_asha
+  metric: test_loss
+  smaller_is_better: true
+  max_trials: 500
+  max_length:
+    epochs: 20
+entrypoint: python3 model_def_adaptive.py
```

Comment:
You still have a bunch of files missing the final newline, please fix them.  If you look at the `Files Changed` tab for the commit message, github identifies missing the newline with a "do not enter"-looking symbol.  Please fix them.